### PR TITLE
Add issue template for creating release checklist issues

### PR DESCRIPTION
**Pull request by cwayne18**
Signed-off-by: cwayne18 <cwayne@ubuntu.com>

#### Proposed Changes ####

Adds an issue template for ensuring release checklist issues are formatted properly

#### Types of Changes ####

New issue template

#### Verification ####

Additional issue template when creating a new issue

### CODE REVIEW

**Review by brandond — APPROVED** [review]

**Review by briandowns — APPROVED** [review]

**Comment by briandowns** [comment]
Should we provide additional links for more detail on some of those? I think that might be helpful.

**Comment by cwayne18** [comment]
> Should we provide additional links for more detail on some of those? I think that might be helpful.

We certainly could! This was just a copy from the most recent "cut release" issue so it's the current process. That said, I'm definitely not against enhancing it, do you have any specific suggestions to add?

**Comment by cwayne18** [comment]
To see what this will look like, you can see: https://github.com/cwayne18/k3s/issues/new/choose

File: .github/ISSUE_TEMPLATE/cut_release.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+**List of required releases:**
+_To release as soon as able for QA:_
+- {VERSION}
+
+_To release once have approval from QA:_
+- {VERSION} (Never release on a Friday unless specified otherwise)
+
+**Prep work:**
+- [ ] PM: Dev and QA team to be notified of the incoming releases - send calendar invite
```
Comment (davidnuzik) [inline .github/ISSUE_TEMPLATE/cut_release.md:23]: Nit:
```suggestion
- [ ] PM: Dev and QA team to be notified of the incoming releases - add event to team calendar
```

File: .github/ISSUE_TEMPLATE/cut_release.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- [ ] PM: Dev and QA team to be notified of the date we will mark the latest release as stable - send calendar invite
```
Comment (davidnuzik) [inline .github/ISSUE_TEMPLATE/cut_release.md:24]: Two things below.

Nit:
```suggestion
- [ ] PM: Dev and QA team to be notified of the date we will mark the latest release as stable - add event to team calendar
```

Note:
Only the latest minor release would have this checklist item (e.g. 1.21.x currently). So, do we need to qualify this or something? Basically this checklist item does not apply to older minor release branches like 1.19 and 1.20 today.

**Review by davidnuzik — CHANGES_REQUESTED** [review]
Requesting couple small changes.

**Review by davidnuzik — APPROVED** [review]
LGTM, can we squash and merge?

**Review by galal-hussein — APPROVED** [review]

**Review by fapatel1 — APPROVED** [review]
LGTM

**Pull request merged**